website/pages/blog/announcing-conductor-alpha-release.mdx

```diff
@@ -0,0 +1,209 @@
+  on what matters most - building great and performant GraphQL applications.
+- **Real open-source**: Conductor is designed to be accessible to everyone, it is open-source (MIT)
+  and free - and will always be. You can self-host it, and you have full access to all the plugins
+  typically reserved for enterprise plans in other solutions. This includes rate limiting, caching,
+  authentication, persisted queries, open telemetry, and federation capabilities.
+- **Supporting Diverse GraphQL Architectures**: Whether you're working with federated, or standalone
+  APIs, Conductor is versatile enough to handle it all. Our goal is to provide a gateway that adapts
+  to all graphql projects' needs.
+- **Community-Driven Development**: As a community-focused organization, The Guild is constantly
+  seeking feedback and contributions. Our roadmap is public! Your insights and suggestions will
+  shape the future of Conductor, making it a tool by the community, for the community.
+
+## Unpacking Conductor's Feature Set
+
+Conductor isn't just another tool; it's a comprehensive solution for managing your GraphQL
+architecture. Here's a glimpse of what it offers:
+
+- **GraphQL Spec Compliance**: Fully adheres to the GraphQL specification, ensuring reliable and
+  standard-compliant behavior.
+- **Advanced Gateway Capabilities**: Serves as a dynamic very thin layer with minimal overhead
+  between GraphQL consumers and servers, enhancing GraphQL runtime with robust plugins for caching,
+  authentication, rate limiting, CORS, persisted queries (trusted documents), and OpenTelemetry.
+- **Distributed Schemas with Experimental Federation Support**: Conductor integrates seamlessly with
+  Apollo Federation, managing all aspects from query planning to response merging. However, it's
+  important to note that the federation query planner in Conductor is still in its experimental
+  phase. Users may encounter certain limitations or issues, and we are actively working on it.
+- **Flexible and Scalable**: Whether you're working on a small project or a large enterprise
+  application, Conductor scales with your needs. Its architecture is modular, and can fully
+  efficiently utlize your hardware resources.
+- **Extensible Endpoint Configuration**: Allows exposure of multiple GraphQL endpoints from a single
+  instance with configurable plugins per endpoint.
+- **VRL (Vector Routing Language) Support**: Offers limitless possibilities for custom logic,
+  plugins, and response transformers.
+- **Comprehensive Security & Monitoring**: Built-in support for various authentication methods,
+  authorization, rate limiting, and OpenTelemetry for monitoring.
+- **Flexible runtime**: Conductor runs either as a binary (and dockerized), and can also run on the
+  Edge (CloudFlare Worker).
+
+## How Does It Work?
+
+Conductor acts as a gateway/proxy between your consumers and your GraphQL server. It intercepts the
```

Comment:
it could also be multiple GraphQL servers? 🤔

Reply:
yeah, we mention that below, I'll get rid of the how does it work section anyways.

https://github.com/the-guild-org/website/pull/1512/files#diff-6195f3407ed0f19351453da2aa08ab2ebff068b75b312f5d5e47e73dcd5cc553R52